Wurstpack/wurstscript/lib/systems/Knockback3.wurst

```diff
@@ -0,0 +1,217 @@
+package Knockback3
+
+import Terrain
+import MapBounds
+import LinkedListModule
+import HashMap
+
```

Comment:
Put example up here, remove // inside hot doc and make destructable callback configurable